bgw-core/src/main/kotlin/tools/aqua/bgw/components/uicomponents/TextField.kt

```diff
@@ -33,8 +33,9 @@ import tools.aqua.bgw.util.Font
  * @param width Width for this [TextField]. Default: [DEFAULT_TEXT_FIELD_WIDTH].
  * @param height Height for this [TextField]. Default: [DEFAULT_TEXT_FIELD_HEIGHT].
  * @param text Initial text for this [TextField]. Default: empty String.
- * @param prompt Prompt for this [TextField]. This gets displayed as a prompt to the user whenever the label is an
- * empty string. Default: empty string.
+ * @param prompt Prompt for this [TextArea].
```

Comment:
Docs should reference TextField instead of TextArea. Might be a C&P error.
Default value ist missing as well. Should be consistent with TextArea.

Reply:
KDoc reference changed to [TextField].